civiform/civiform
# Add program image alt-text description to admin UI & database

### Description

On the "Manage program image" page, adds a text to set a description for the image and save it to the program details.

Note: The UX hasn't been finalized yet, so this PR doesn't have any nice styling. That styling will be added once the UX is final.

## Release notes

### Checklist

#### General


#### User visible changes


#### New Features


### Instructions for manual testing

1. Enable PROGRAM_CARD_IMAGES flag.
2. On a draft program, click "Edit program image".
3. Verify the page has a text box to fill in the image description.
4. Fill in a description and click Save -> verify a success toast appears and your new description is present in the text box.

### Issue(s)
Fixes #5997 

Epic: #5676
